README.md

```diff
@@ -1,2 +1,14 @@
-# beyond-os-06-group-2
-beyond-os-06-group-2
+# Falcon
+
+Manage and control workflow app
+
+# Installation
+
+1. Install [Vagrant](https://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/)/[VMWare](https://www.vmware.com/) hypervisor.
+2. Clone [beyond-os-06-group-2](https://github.com/beyond-io/beyond-os-06-group-2) repository.
+   ```sh
+   https://github.com/redhat-beyond/beyond-os-06-group-2
+   ```
+3. Open any terminal and navigate to the project directory.
+4. Run the `vagrant up` command.
+5. Use any browser and navigate to - `localhost:8080`.
```

Comment:
Not sure if I understood the concepts the right way, but isn't it should be `localhost:8000`?
Same port as the guest and host Vagrantfile

Reply:
> Not sure if I understood the concepts the right way, but shouldn't it be `localhost:8000`? Same port as the guest and host on Vagrantfile

Good catch @orzionpour